v4/Policies/SurveyPolicy.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace v4\Policies;
+use Ushahidi\App\Auth\GenericUser;
+use Ushahidi\App\Auth\GenericUser as User;
+use Ushahidi\Core\Entity;
+use v4\Models\Survey;
+use Ushahidi\Core\Entity\Permission;
+use Ushahidi\Core\Traits\AdminAccess;
+use Ushahidi\Core\Traits\UserContext;
+use Ushahidi\Core\Traits\ParentAccess;
+use Ushahidi\Core\Traits\PrivAccess;
+use Ushahidi\Core\Traits\PrivateDeployment;
+use Ushahidi\Core\Tool\Permissions\AclTrait;
+
+class SurveyPolicy
+{
+
+    // The access checks are run under the context of a specific user
+    use UserContext;
```

Comment:
I dont love this but wanted to keep this looking as similar as our authorizer code as possible and reuising almost everything we do to limit the possibility of regression especially around data privacy/permissions ! it's a decent workaround during Buffy but something to rework heavily in Origami so we stop this bit of duplication